src/Model/Table/UsersTable.php

```diff
@@ -592,4 +592,13 @@ public function updatePasswordVersion($userId, $plainTextPassword)
             $this->save($user);
         }
     }
+
+    public function updateLastContribution($userId)
+    {
+        if ($this->exists(["Users.id" => $userId])) {
```

Comment:
You can use the following pattern instead:
```php
use Cake\Datasource\Exception\RecordNotFoundException;

try {
    $user = $this->get($userId);
} catch (RecordNotFoundException $e) {
}
```
The benefit is that only one SQL SELECT request is performed (at `get()`) instead of two (both at `exists()` and `get()`).